### DISCUSSION TITLE
Fuel costs receipts table from EIA 923 projections (from Slack)

### DISCUSSION THREAD

**Discussion by bendnorman** [catalyst-cooperative/pudl#3682] [Help Me!] [answered]
For the fuel costs receipts table from EIA 923, do you do any projections on fuel costs or is it straight from the EIA form?

**Comment by bendnorman** [comment, marked as answer]
There are some values which are filled in from the state level monthly aggregates published in the EIA bulk electricity data, but the overwhelming majority are just directly reported.   There’s a boolean column indicating which values are filled.  We’ve just added fuel price projections from the most recent EIA AEO. They should appear in tonight’s nightly build outputs if the builds succeed.

Note, this is for the unaggregated table `out_eia923__fuel_receipts_costs`  — there are also yearly and monthly aggregated tables with far fewer columns which sum up all the fuel deliveries of a particular fuel type, and provide the weighted average heat content and price per unit, but also lose the delivery-specific information like supplier and transportation method.

**Reply by zaneselvans** [reply]
Another potentially important distinction in this table is that these are fuel **deliveries** not fuel consumption, so it's possible that the fuel deliveries could fluctuate significantly and diverge from the monthly consumption, especially for like, solid and liquid fuel where there's a lot of on-site storage capacity.
